dbms/src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -2528,6 +2541,17 @@ void MergeTreeData::swapActivePart(MergeTreeData::DataPartPtr part_copy)
 
             auto part_it = data_parts_indexes.insert(part_copy).first;
             modifyPartState(part_it, DataPartState::Committed);
+
+            Poco::Path marker_path(Poco::Path(original_active_part->getFullPath()), DELETE_ON_DESTROY_MARKER_PATH);
+            try
+            {
+                Poco::File(marker_path).createFile();
+            }
+            catch (...)
+            {
+                LOG_WARNING(log, "Exception has occurred while creating DeleteOnDestroy marker: '"
```

Comment:
Need to log information about exception.

Reply:
+